### DISCUSSION TITLE
Training planner?

### DISCUSSION THREAD

**Discussion by Gobolino** [ho-dev/HattrickOrganizer#1819] [Ideas]
HI !!!, there's a feature in hattrick portal called "Training planner" 
( https://hattrickportal.pro/Utils/PlayerTrainingPlan.aspx )
The visual aspect of HO would be great to implement this, I HOPE. 
I was kinda wondering if there's a chance of it being shown PER PLAYER.

**Comment by wsbrenk** [comment]
the training module of the HO should allow a much more detailed training planning of your team. The development of a single selected player is also displayed in detail:

![image](https://user-images.githubusercontent.com/11821502/224474191-42f9cc5f-0514-463b-9de7-ab50ab05f845.png)

![image](https://user-images.githubusercontent.com/11821502/224474242-33856d0f-fb30-465a-8223-e6bec805a42d.png)

**Comment by Gobolino** [comment]
Thanks!, I was thinking in the lines of:
For example, I have these 6 players that I want them to be a forward...

IF I understand HO as it is right now, it means: 

First player, I change the future training for passing until he reaches X and scoring until he reaches Y and Pm until Z... ok, perfect... 
Now I look at the second play... damn, he falls short in everything but Pm... ok, how about 1 extra week of passing... good!, he made it, now scoring... ok, 2 weeks. 
Now I look at the 3rd player... oh, he is good but lacks pm, ok... 3 weeks for Pm....
Damn, if I add 3 weeks for pm now, the first player will lack passing and scoring when reaching that date, ok... so I'll compensate that with 3 extra weeks of one and 2 weeks for the other... ok, looks good. 
Now the 4th player... is good, ok. 
5th one, damn, he needs 2 more weeks of scoring... now where the hell I put them now... ok, at the end so I don't mess with anyone else. 
Ah, the 6th one needs passing.... but if I put passing at the end....

The "idea" (which I don't know anything about coding so it maybe be impossible to do anyway) was:
Select these 6 players, I want them all to have these X, Y and Z stats in these categories... I need the best training routine for the next weeks, click.

**Comment by wsbrenk** [comment]
good idea
